### ISSUE TITLE
1.16.5

### ISSUE THREAD

**Issue by mrGRINCH** [Queerbric/Inspecio#2]
Any chance of getting this updated for 1.16.5?

**Comment by LambdAurora** [comment]
Sadly, Inspecio uses a new system first introduced in Minecraft 1.17 snapshots. Reimplementing it on 1.16.5 would require a lot of time and effort that just isn't worth it.

**Comment by cplir-c** [comment]
same response as [last time someone asked](https://github.com/Queerbric/Inspecio/issues/1)

**Comment by mrGRINCH** [comment]
Ah I see, thanks for the reply.

**Comment by srnyx** [comment]
> Sadly, Inspecio uses a new system first introduced in Minecraft 1.17 snapshots. Reimplementing it on 1.16.5 would require a lot of time and effort that just isn't worth it.

Know any good alternatives?

**Comment by GamingInfinite** [comment]
unfortunate, but can't wait until 1.17 is mainstream 😄

**Comment by LambdAurora** [comment]
> Why is this still pinned?

You managed to confuse me for a second, oops.

tl;dr pinned issues that are closed are still a good way to answer quickly to a question someone might want to ask in the issue tracker.
